### PR TITLE
Add support to parse and load device MAC addresses from Device MfgInfo

### PR DESCRIPTION
**Pull request by shrikant1407**
Add support to parse and load device MAC addresses from Device Mfg Info

### CODE REVIEW

**Review by DukeDavis12 — APPROVED** [review]

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+public class ManufacturingInfoStorageFunction {
+
+  /**
+   * Stores TPM EK Certificate into database.
+   * @param serialNo Device serial number that is used to retrieve TPM EK Data.
+   * @param macAddresses MAC addresses received from the client.
+   * @throws IOException Throws exception if required mfgVoucher is null.
+   */
+  public void store(String serialNo, byte[] macAddresses) throws IOException {
+    Session session = HibernateUtil.getSessionFactory().openSession();
```
Comment (DukeDavis12) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java:31]: If required, we can a check here.

**Review by Sai-Anudeep47 — APPROVED** [review]

**Review by DukeDavis12 — DISMISSED** [review]

**Review by SSWAMIN1SSS — APPROVED** [review]

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.fidoalliance.fdo.protocol.dispatch.VoucherStorageFunction;
+import org.fidoalliance.fdo.protocol.entity.ManufacturedVoucher;
+import org.fidoalliance.fdo.protocol.message.ManufacturingInfo;
+import org.fidoalliance.fdo.protocol.message.OwnershipVoucher;
+import org.fidoalliance.fdo.protocol.message.OwnershipVoucherHeader;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+/**
+ * Stores TPM EK Certificate into database.
```
Comment (shrikant1407) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java:20]:
```suggestion
 * Stores MAC Addresses into database.
```

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ */
+public class ManufacturingInfoStorageFunction {
+
+  /**
+   * Stores TPM EK Certificate into database.
```
Comment (shrikant1407) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java:25]:
```suggestion
   * Stores MAC Addresses into database.
```

File: protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+   * @param serialNo Device serial number that is used to retrieve TPM EK Data.
```
Comment (shrikant1407) [inline protocol/src/main/java/org/fidoalliance/fdo/protocol/db/ManufacturingInfoStorageFunction.java:26]:
```suggestion
   * @param serialNo Device serial number that is used to retrieve MAC Addresses.
```

**Review by DukeDavis12 — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**